js_modules/dagit/packages/core/src/testing/ApolloTestProvider.tsx

```diff
@@ -22,27 +22,35 @@ export interface ApolloTestProps {
    * objects.
    */
   mocks?: any;
+
+  resolvers?: any;
 }
 
 interface Props extends ApolloTestProps {
   typeDefs: DocumentNode;
 }
 
 export const ApolloTestProvider: React.FC<Props> = (props) => {
-  const {children, mocks = [], typeDefs} = props;
+  const {children, mocks = [], typeDefs, resolvers = []} = props;
 
   const client = React.useMemo(() => {
     const mocksToMerge = Array.isArray(mocks) ? mocks : [mocks];
     const mocksWithMerge = mergeResolvers([defaultMocks, ...mocksToMerge]);
 
     const schema = makeExecutableSchema({typeDefs});
-    const mockedSchema = addMocksToSchema({schema, mocks: mocksWithMerge});
+    const mockedSchema = addMocksToSchema({
+      schema,
+      mocks: mocksWithMerge,
+      resolvers: () => {
+        return mergeResolvers(resolvers);
+      },
```

Comment:
Can this just be `resolvers: mergeResolvers(resolvers)`?

Reply:
it has to be a function. It takes in `store` as an argument but offline we agreed we probably don't want tests to do anything with the store since that would equate to reimplementing the server and we don't want that in our tests